src/NuGet.Clients/NuGet.CommandLine/Commands/InstallCommand.cs

```diff
@@ -156,13 +156,21 @@ private Task PerformV2Restore(string packagesConfigFilePath, string installPath)
                     reference,
                     new[] { packagesConfigFilePath },
                     isMissing: true));
+
+            var packageSources = GetPackageSources(Settings);
+            Console.WriteLine("Feeds used:");
+            foreach (var packageSource in packageSources)
+            {
+                Console.WriteLine(packageSource.Source);
+            }
+
             var packageRestoreContext = new PackageRestoreContext(
                 nuGetPackageManager,
                 packageRestoreData,
                 CancellationToken.None,
                 packageRestoredEvent: null,
                 packageRestoreFailedEvent: null,
-                sourceRepositories: GetPackageSources(Settings).Select(sourceRepositoryProvider.CreateRepository),
+                sourceRepositories: packageSources.Select(sourceRepositoryProvider.CreateRepository),
```

Comment:
Addressed by changing GetPackageSources to return an `IReadOnlyCollection` instead of `IEnumerable`. No need to call `ToList` on it, as the method constructs a `List` internally. https://github.com/NuGet/NuGet.Client/commit/81bcb3ec8f71a66a10007a309b591db60d39498d